Redis: adding annotation and service type to redis setup

kavehmzta opened the pull request:
Adding annotations and service type for redis chart.

kavehmz committed:
Adding service type and service annotations to redis

kavehmz committed:
Merge remote-tracking branch 'up/master' into redis

kavehmz committed:
Merge remote-tracking branch 'origin/master' into redis

mattfarina commented on stable/redis/values.yaml:
```diff
@@ -87,3 +87,7 @@ networkPolicy:
+service:
+  type: ClusterIP
+  annotations: {}
+  loadBalancerIP:
```
According to the linter:

>  93:18     error    no new line character at the end of file  (new-line-at-end-of-file)

Please add a new line at the end of the file.

kavehmz committed:
Merge remote-tracking branch 'up/master' into redis

kavehmz committed:
redis: bumping the redis version

unguiculus commented:
/ok-to-test

unguiculus commented:
Please add documentation to the readme.

kavehmz committed:
Merge remote-tracking branch 'up/master' into redis

kavehmz committed:
[stable/redis]
   Adding service.annotations and service.loadBalancerIP to readme. I see mix naming conventions that is confusing but still I try to do the change in a backward compatible manner

kavehmz committed:
[stable/redis] bumping the version to 1.0.3

unguiculus commented:
/lgtm
